gpu/test_gpu.py

```diff
@@ -127,12 +131,13 @@ def test_install_gpu_cuda_nvidia(self, configuration, machine_suffixes,
         timeout_in_minutes=30,
         boot_disk_size="50GB")
     for machine_suffix in machine_suffixes:
-      self.verify_instance("{}-{}".format(self.getClusterName(),
-                                          machine_suffix))
+      machine_name="{}-{}".format(self.getClusterName(),machine_suffix)
+      self.verify_instance(machine_name)
+      self.verify_instance_nvcc(machine_name, cuda_version)
 
   @parameterized.parameters(
-      ("STANDARD", ["m"], GPU_H100, None, "NVIDIA", "12.4"),
-      ("STANDARD", ["m"], GPU_A100, None, "NVIDIA", "11.8"),
+      ("STANDARD", ["m"], GPU_H100, GPU_A100, "NVIDIA", "12.4"),
```

Comment:
If we update to RAPIDS 24.08, then would suggest changing this to CUDA 12.5

Though if there is some other limitation going on (like hardware, installed driver, etc.), using whatever CUDA 12.x version the machine supports makes sense

```suggestion
      ("STANDARD", ["m"], GPU_H100, GPU_A100, "NVIDIA", "12.5"),
```

Reply:
can we do that change next time?  I can try switching it now, but I've got so much aligned with 12.4 right now that I'm afraid I'd lose a lot of time if we bumped it up again.

Reply:
Sounds good. Let's follow up after